src/scheduler/gc_work.rs

```diff
@@ -381,9 +381,9 @@ pub trait ProcessEdgesWork:
         if Self::SCAN_OBJECTS_IMMEDIATELY {
             // We execute this `scan_objects_work` immediately.
             // This is expected to be a useful optimization because,
-            // say for _pmd_ with 200M heap, we're likely to have 50000~60000 `ScanObjects` works
+            // say for _pmd_ with 200M heap, we're likely to have 50000~60000 `ScanObjects` work
             // being dispatched (similar amount to `ProcessEdgesWork`).
-            // Executing these works now can remarkably reduce the global synchronization time.
+            // Executing these work now can remarkably reduce the global synchronization time.
```

Comment:
this work?

Reply:
I will just use 'these work packets' to avoid possible mistake with work plural.